spec/threema/receive/delivery_receipt_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe Threema::Receive::DeliveryReceipt do
+  subject { described_class.new(**attributes) }
+  let(:attributes) { attributes_for(:delivery_receipt_receive) }
```

Comment:
I don't know if `attributes` is the best naming here. I think it's more like `payload`, right? But if it's already called like that in the other specs, I'm fine with it.

Reply:
yes, and it uses the `attributes_for` method from **FactoryBot**. From the documentation, this method:

```
 Generates a hash of attributes for a registered factory by name.
```

will leave as is.